### PR TITLE
{tools}[GCCcore/10.2.0] configurable-http-proxy v4.2.1, nodejs v12.19.0, JupyterHub v1.1.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
depends on ~~#11536~~ (for Mako)
I intentionally didn't follow the old configs naming convention for JupyterHub

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@Micket Should we rename the (ancient) `jupyterhub` easyconfigs to avoid having the same thing under two different names?

**Review by boegel — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/c25f29a07a91504386311e7d34b56706 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in total)
node3109.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c75a7666553d8b46b91414e0b93555b0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/13ac98c9deec3d1571e06a577e86e175 for a full test report.

**Comment by Micket** [comment]
I think this is good to merge @boegel

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/372bebe9a59bb0f0bdb77f9bfe6670ac for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

File: easybuild/easyconfigs/n/nodejs/nodejs-12.19.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://%(name)s.org/dist/v%(version)s/']
+sources = ['node-v%(version)s.tar.gz']
+checksums = ['1279e49be60d92cf4c1a48c9d92397db4e9284a100bc47689ce7924686bbddd1']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Python', '3.8.6'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/nodejs/nodejs-12.19.0-GCCcore-10.2.0.eb:21]: Apparently `nodejs` *still* requires `python2` at configure time... Fixed in #11679

**Pull request merged**